### PR TITLE
Password reset.

### PR DESCRIPTION
**Pull request by Gastonius**
Fully tested on Chrome / App.

### CODE REVIEW

File: chrome_extension/vendor/mooltipass/device.js [hunk 1]
```diff
@@ -596,6 +590,25 @@ mooltipass.device.messageListener = function(message, sender, sendResponse) {
...
+ * 
+ */
+mooltipass.device.checkInterference = function() {
+    if (page.settings.useMoolticute) { // Use Moolticute
+        return;
+    } else { // Use APP
+            var moolticute_connection = new WebSocket('ws://127.0.0.1:30035');
+            moolticute_connection.onopen = function(evt) {
+                // Moolticute is open!!
+                console.error('MOOOLTICUTE IS OPEN!!');
```
Comment (limpkin) [inline chrome_extension/vendor/mooltipass/device.js]: for a next PR: maybe change the extension icon and send a notification?

Comment (Gastonius) [inline chrome_extension/vendor/mooltipass/device.js]: Yes, I'm sorry, that's actually part of a different commit / Work in progress

Comment (Gastonius) [inline chrome_extension/vendor/mooltipass/device.js]: Won't hurt to be here tho... so I don't think we need to roll back

**Comment by limpkin** [comment]
yeah that's alright.

**Pull request merged**